backends/gdx-backend-android/src/com/badlogic/gdx/backends/android/keyboardheight/KeyboardHeightObserver.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * This file is part of Siebe Projects samples.
+ *
+ * Siebe Projects samples is free software: you can redistribute it and/or modify
+ * it under the terms of the Lesser GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Siebe Projects samples is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * Lesser GNU General Public License for more details.
+ *
+ * You should have received a copy of the Lesser GNU General Public License
+ * along with Siebe Projects samples.  If not, see <http://www.gnu.org/licenses/>.
+ */
```

Comment:
Is libGDX's license actually compatible with this code?

Reply:
It depends on the copyrightability of interfaces. (IANAL) The US Supreme Court appeared to rule in favor of Google in Oracle v. Google, where Oracle's argument was that its interfaces were its property and copyright Oracle, and Google's argument was that an interface specifies a calling convention and does not constitute code. So, the highest court in my country (where GitHub is incorporated) says it is compatible, but I have no idea what it says in Austria.